.env.development

```diff
@@ -1 +1 @@
-REACT_APP_API_URL=https://360ApiTrain.gordon.edu/
+REACT_APP_API_URL=https://360ApiTrain.gordon.edu/
```

Comment:
Did you mean to delete the end-of-line character?  It's probably better to have it, unless it caused some problem.

Reply:
Change made

Reply:
Change made